src/seedu/addressbook/AddressBook.java

```diff
@@ -989,16 +1001,23 @@ private static String extractPhoneFromPersonString(String encoded) {
 
         // phone is last arg, target is from prefix to end of string
         if (indexOfPhonePrefix > indexOfEmailPrefix) {
-            return removePrefixSign(encoded.substring(indexOfPhonePrefix, encoded.length()).trim(),
-                    PERSON_DATA_PREFIX_PHONE);
+            return removePrefixPhone(encoded.substring(indexOfPhonePrefix, encoded.length())); 
 
         // phone is middle arg, target is from own prefix to next prefix
         } else {
-            return removePrefixSign(
-                    encoded.substring(indexOfPhonePrefix, indexOfEmailPrefix).trim(),
-                    PERSON_DATA_PREFIX_PHONE);
+            return removePrefixPhone(encoded.substring(indexOfPhonePrefix, indexOfEmailPrefix).trim());
```

Comment:
The main goal of this activity is to change the `removePrefixSign` method. Please check the instruction again.